src/ng_package/angular_package_format.bzl

```diff
@@ -106,16 +106,15 @@ def _run_rollup(ctx, rollup_config, inputs, dts_mode):
     # bazel rule prints nothing on success.
     args.add("--silent")
 
-    other_inputs = [rollup_config]
+    other_inputs = [rollup_config] + ctx.files.rollup_runtime_deps
```

Comment:
This is where we should place a TODO right?

Reply:
Added it in both places.